content/5_Plat/Sparse_Segtree.mdx

```diff
@@ -235,7 +236,8 @@ class SparseSegtree {
 int main() {
 	int query_num;
 	cin >> query_num;
-	SparseSegtree st((int)(1e9 + 1), query_num);
+	const int range_size = 1e9 + 1;
```

Comment:
is the range inclusive or exclusive

Reply:
like all the other ds impls in the guide it's $[0, n)$ where $n$ is the size given